cpp/tests/search/search_test.cu

```diff
@@ -1550,3 +1551,207 @@ TEST_F(SearchTest, contains_nullable_column_false_string)
 
   ASSERT_EQ(result, expect);
 }
+
+TEST_F(SearchTest, multi_contains_some)
+{
+  using element_type = int64_t;
+
+  fixed_width_column_wrapper<element_type>    column {0, 1, 17, 19, 23, 29, 71};
+  fixed_width_column_wrapper<element_type>    in_col {17, 19, 45, 72};
+
+  fixed_width_column_wrapper<bool8>           expect {0, 0, 1, 1, 0, 0, 0};
+
+  auto result = cudf::experimental::contains(column, in_col);
+
+  expect_columns_equal(*result, expect);
+}
+
+TEST_F(SearchTest, multi_contains_none)
+{
+  using element_type = int64_t;
+
+  fixed_width_column_wrapper<element_type>    column {0, 1, 17, 19, 23, 29, 71};
+  fixed_width_column_wrapper<element_type>    in_col {2, 3};
+
+  fixed_width_column_wrapper<bool8>           expect {0, 0, 0, 0, 0, 0, 0};
+
+  auto result = cudf::experimental::contains(column, in_col);
+
+  expect_columns_equal(*result, expect);
+}
+
+TEST_F(SearchTest, multi_contains_some_string)
+{
+  std::vector<const char*> h_col_strings    { "0", "1", "17", "19", "23", "29", "71"};
+  std::vector<const char*> h_in_col_strings { "17", "19", "45", "72" };
+
+  cudf::test::strings_column_wrapper column(h_col_strings.begin(),
+                                            h_col_strings.end());
+
+  cudf::test::strings_column_wrapper in_col(h_in_col_strings.begin(),
+                                            h_in_col_strings.end());
+
+  fixed_width_column_wrapper<bool8>           expect {0, 0, 1, 1, 0, 0, 0};
+
+  auto result = cudf::experimental::contains(column, in_col);
```

Comment:
Hang on, does your hash table work for strings?

Reply:
Yes